opticrd/opticrd.github.io
# Updated the centers list

#### Description:

Updated the vaccination centers list.

#### Linked issues:

[Actualizar base de datos de centros de 2da dosis. ](https://trello.com/c/Z6IQozLk/82-actualizar-base-de-datos-de-centros-de-2da-dosis)

#### How to test:

1. Open the index.html file in a browser and verify that the centers have been updated.
